scripts/affected_targets

```diff
@@ -14,43 +14,78 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Finds all targets of a given kind affected by changes since origin/develop.
+# Finds all targets of a given kind affected by changes in a given commit range.
 
-# type should be a bazel `kind` type. E.g. test or rule.
+SCRIPT_NAME=$(basename "$0")
+
+# bazel_kind should be a bazel `kind` type. E.g. test or rule.
 # https://docs.bazel.build/versions/master/query.html#kind
-type="$1"
+bazel_kind="$1"
+
+# range should be a valid git diff/log range. E.g. origin/develop...HEAD
+range="$2"
+
+if [ -z "$bazel_kind" ]; then
+  echo "Please provide a blaze rule kind to query for."
+  exit 1
+fi
+
+if [ -z "$range" ]; then
+  echo "Please provide a diff range."
+  exit 1
+fi
+
+if [ $(git branch --list "$TARGET_BRANCH") ]; then
+  base_sha=$(git merge-base "$TARGET_BRANCH" HEAD)
+else
+  base_sha="HEAD^"
+fi
 
 # Generates a list of files that were modified since origin/develop.
```

Comment:
Tip/nit: "Modified within $range" or something like that?

Reply:
https://github.com/material-components/material-components-ios/pull/6094